packages/components/src/mobile/image/index.native.js

```diff
@@ -80,6 +83,27 @@ const ImageComponent = ( {
 					onImageDataLoad( metaData );
 				}
 			} );
+
+			if ( url.startsWith( 'file:///' ) ) {
+				setLocalURL( url );
+			} else if ( url.startsWith( 'https://' ) ) {
+				if ( Platform.isIOS ) {
+					setNetworkURL( url );
+				} else if ( Platform.isAndroid ) {
+					RNImage.prefetch( url ).then(
+						() => {
+							if ( ! isCurrent ) {
+								return;
+							}
+							setNetworkURL( url );
+							setNetworkImageLoaded( true );
+						},
+						() => {
+							// handle error
```

Comment:
If we do not intend to handle this error, we should update this inline comment stating why the error is not handled or remove the no-op callback.

Reply:
Updated with a code comment that this UI state is handled by `isUploadFailed` event. Ignoring the error here will persist the local URI in the UI.